lib/registrant.rb

```diff
@@ -0,0 +1,22 @@
+
+class Registrant 
+    attr_reader :name, :age, :permit, :license_data
```

Comment:
Careful here, we do not want `:permit` in the attr_readers because we don't want a method called `permit`. We have a method called `permit?` that acts as an attr_reader for us.